locksmith/__tests__/controllers/v2/verifierController.test.ts

```diff
@@ -1,86 +1,95 @@
+import { ethers } from 'ethers'
 import request from 'supertest'
 import { getWalletInput } from '../../test-helpers/utils'
 
 const app = require('../../../src/app')
 
 jest.setTimeout(600000)
+
 const lockAddress = '0x3F09aD349a693bB62a162ff2ff3e097bD1cE9a8C'
+const network = 4
+jest.mock('@unlock-protocol/unlock-js', () => {
+  return {
+    Web3Service: jest.fn().mockImplementation(() => {
+      return {
+        isLockManager: (lock: string) => lockAddress === lock,
+      }
+    }),
+  }
+})
 
 describe('Verifier v2 endpoints for locksmith', () => {
-  it('Add verifier to list and not add duplicates', async () => {
-    expect.assertions(3)
-    const network = 4
+  it('Get list items without authorization', async () => {
+    expect.assertions(1)
+    const lock = await ethers.Wallet.createRandom().getAddress()
+    const getListEndpoint = await request(app).get(
+      `/v2/api/verifier/${network}/${lock}`
+    )
+    expect(getListEndpoint.status).toBe(403)
+  })
 
-    const {
-      walletAddress: address,
-      message,
-      signedMessage,
-    } = await getWalletInput()
+  it('Get list items from lock with random address (not lockManager)', async () => {
+    expect.assertions(2)
+    const lock = await ethers.Wallet.createRandom().getAddress()
+    const { message, signedMessage } = await getWalletInput()
```

Comment:
Ok you can use `loginRandomUser` from [test-helpers/utils.ts](https://github.com/unlock-protocol/unlock/commit/4bce4c2baf3d7f05ae873115250133eb38d587c7#diff-a56a299a69adfb724d9c98bdffe39f8a43c136d2b514d1f442ad48cc662033f6)

Reply:
oh thk! done!